test/blob_test.js

```diff
@@ -1,4 +1,4 @@
-suite.only("Azure Blob", function() {
+suite("Azure Blob", function() {
```

Comment:
In test 'delete container with if-modified-since conditional header', I was creating a container and after that I wanted to deleted it using the conditional header 'if-modified-since' set to now() date. I was expecting to fail because lastModified < now(), but sometimes lastModified was equals with now() and because of this I put a small sleep there. This fixes the test.